### ISSUE TITLE
Used the pre-trained model model_image_zooms_2, it returns action 6 on the very first prediction with similar airplane test images

### ISSUE THREAD

**Issue by sandipan** [imatge-upc/detection-2016-nipsws#31]
Used the pre-trained model model_image_zooms_2, it returns action 6 on the very first prediction with similar airplane test images. I am getting qval returned like the following one: [[ 0.00517994  0.00194971  0.00551834  0.00129612 -0.00100664  0.02811333]] on the very first prediction with the pretrained model.  Input images are preprocessed as expected (mean normalized and converted to BGR). Getting no clue where I am doing wrong. Any idea where things can go wrong / how to debug? thanks in advance.
